source/includes/language-compatibility-table-pymongo.rst

```diff
@@ -91,6 +111,7 @@ Python 3
      -
 
    * - 3.13
```

Comment:
Tech reviewer: This table says v3.13 of the driver supports CPython 3.11. Is that accurate?

Reply:
Yes, that is accurate